### PR TITLE
fix basic config load issue, register log filtering

### PR DESCRIPTION
**Pull request by vishalya**

### CODE REVIEW

**Review by andythsu — APPROVED** [review]
LGTM

File: gateway-ha/gateway-ha-config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ requestRouter:
   port: 8080
   name: trinoRouter
   historySize: 1000
-  requestBufferSize: 8 * 1024
+  requestBufferSize: 8195
```
Comment (rdsarvar) [inline gateway-ha/gateway-ha-config.yml:9]: question: how come we opted into 8195 instead of 8192?

Comment (vishalya) [reply gateway-ha/gateway-ha-config.yml]: It's a typo, thanks for catching it.

**Review by mosabua — APPROVED** [review]
Looks good. I will squash and merge

**Pull request merged**